lib/containers/utils.pm

```diff
@@ -22,7 +22,7 @@ use strict;
 use warnings;
 use version_utils;
 
-our @EXPORT = qw(test_seccomp basic_container_tests set_up get_vars build_img test_built_img);
+our @EXPORT = qw(test_seccomp basic_container_tests set_up get_vars build_img test_built_img is_sle_container_supported);
```

Comment:
I would say this function is more suitable for `version_utils`

Reply:
The reason that i did not add  it to the `version_utils` is that there the conditions are specific to the SUT based on the job variables. I think this is a "special" situation because it is container specific. WDYT?

Reply:
i am resolving this as per our discussion.